blockindex/bloomfilterindexer.go

```diff
@@ -190,18 +189,17 @@ func (bfx *bloomfilterIndexer) BlockFilterByHeight(height uint64) (bloom.BloomFi
 	if err != nil {
 		return nil, err
 	}
-	return bloom.BloomFilterFromBytes(bfBytes)
-}
-
-// RangeFilterByHeight returns the range bloomfilter for the height
-func (bfx *bloomfilterIndexer) RangeFilterByHeight(height uint64) (bloom.BloomFilter, error) {
-	br, err := bfx.rangeBloomFilter(height)
+	bf, err := bloom.NewBloomFilter(bfx.bfSize, bfx.bfNumHash)
 	if err != nil {
 		return nil, err
 	}
-	return br.BloomFilter, nil
+	if err := bf.FromBytes(bfBytes); err != nil {
+		return nil, err
+	}
+	return bf, nil
 }
 
+// TODO: add pagination in bloomFitler
```

Comment:
`// TODO (liuhaai): add pagination in bloomFitler`

Reply:
ok. Will assign a new issue to myself